src/routes/Devilstable.svelte

```diff
@@ -71,45 +64,101 @@
 		}
 		return fractionData;
 	});
+	/** @type {{label: string, values: boolean[]}[]} */
 	let boolData = $derived([
-		...noFractions.map((d) => {
-			/** @type {boolean[]} */ const values = new Array(DATA_MAX).fill(false);
+		...inputChipValueLabels.map((d) => {
+			const info = data.find((c) => c.label === d);
+			if (info) {
+				/** @type {boolean[]} */ const values = new Array(DATA_MAX).fill(false);
+				info?.values.forEach(([start, end]) => {
+					for (let i = start; i <= end; i++) {
```

Comment:
For better readability (and performance?) consider using `array.fill(false, start, end);` instead of for-loop (maybe +/- 1 correction needed)